full-service/src/service/receipt.rs

```diff
@@ -433,6 +433,7 @@ mod tests {
                     subaddress_index: None,
                 },
                 None,
+                None,
```

Comment:
Not testing whether receipts were constructed with txos selected only from a specified subaddress, as this is not functionality specific to receipts.